src/main/resources/templates/edit-customer.html

```diff
@@ -1,46 +1,62 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
 <head>
-  <meta charset="utf-8" />
-  <title>Edit Customer</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css"
```

Comment:
Bootstrap - great to see it. 
One note, since you are using it, current version is 5.3. the 4.X supported (meaning has security patches) is 4.6. 
It is not critical, but keep it in mind or your actual project.